tests/Php83/Php83Test.php

```diff
@@ -73,8 +74,9 @@ public function testInvalidOptionsProvided(int $depth, int $flags, string $expec
     public static function invalidOptionsProvider(): iterable
     {
         yield [0, 0, 'json_validate(): Argument #2 ($depth) must be greater than 0'];
-        yield [\PHP_INT_MAX, 0, 'json_validate(): Argument #2 ($depth) must be less than 2147483647'];
-
+        if(\PHP_INT_MAX > 2147483647){
```

Comment:
Please just run a CS fixer here or apply this patch.
```suggestion
        if (\PHP_INT_MAX > 2147483647) {
```